x-pack/platform/plugins/shared/lens/public/visualizations/datatable/components/constants.ts

```diff
@@ -7,11 +7,28 @@
 
 import { RowHeightMode } from '../../../../common/types';
 
+/**
+ * Row height might be a value from -1 to 20
+ * A value of -1 automatically adjusts the row height to fit the contents.
+ * A value of 1 displays the content in a single line.
+ * A value from 1 to 20 represents number of lines of Document explorer row to display.
+ */
+export const ROWS_HEIGHT_OPTIONS = {
+  auto: -1,
+  single: 1,
+  default: 3,
+} as const;
```

Comment:
nit: maybe an `enum` can suit best in this case

```suggestion
export enum ROWS_HEIGHT_OPTIONS {
  auto = -1,
  single = 1,
  default = 3,
};
```